server_addon/fusion/server/settings.py

```diff
@@ -105,6 +109,18 @@ class FusionSettings(BaseSettingsModel):
                 "farm_rendering"
             ],
             "image_format": "exr"
+        },
+        "CreateImageSaver": {
+            "temp_rendering_path_template": "{workdir}/renders/fusion/{product[name]}/{product[name]}.{ext}",
```

Comment:
This issue here https://github.com/ynput/OpenPype/pull/6057#pullrequestreview-1802621019 might be caused by the wrong template from here? But the template looks ok to me.

Reply:
Most likely resolved by https://github.com/ynput/OpenPype/pull/6057#discussion_r1441566077